Packs/Carbon_Black_Cloud_Enterprise_EDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.py

```diff
@@ -0,0 +1,534 @@
+
+    def device_background_scan_request(self, device_id: Union[list, str]):
+        suffix_url = f'/appservices/v6/orgs/{CB_ORG_KEY}/device_actions'
+
+        body = {
+            'action_type': 'BACKGROUND_SCAN',
+            'device_id': device_id,
+            'options': {
+                'toggle': 'ON'
+            }
+        }
+
+        return self._http_request('POST', suffix_url, json_data=body, resp_type='content')
+
+    def device_background_scan_request_stop(self, device_id: Union[list, str]):
+        suffix_url = f'/appservices/v6/orgs/{CB_ORG_KEY}/device_actions'
+
+        body = {
+            'action_type': 'BACKGROUND_SCAN',
+            'device_id': device_id,
+            'options': {
+                'toggle': 'OFF'
+            }
+        }
+
+        return self._http_request('POST', suffix_url, json_data=body, resp_type='content')
+
+    def device_policy_update(self, device_id: Union[list, str], policy_id: str):
+        suffix_url = f'/appservices/v6/orgs/{CB_ORG_KEY}/device_actions'
+
+        body = {
+            'action_type': 'UPDATE_POLIC',
+            'device_id': device_id,
+            'options': {
+                'policy_id': policy_id
+            }
+        }
+
+        return self._http_request('POST', suffix_url, json_data=body, resp_type='content')
+
+
+def test_module(client):
+    """
+    Returning 'ok' indicates that the integration works like it is supposed to. Connection to the service is successful.
+
+    Args:
+        client: HelloWorld client
+
+    Returns:
+        'ok' if test passed, anything else will fail the test.
+    """
+    client.test_module_request()
+    return 'ok'
+
+
+def alert_list_command(client: Client, args: dict):
+
+    group_results = args.get('group_results')
+    minimum_severity = args.get('minimum_severity')
+    create_time = {
+        'start': args.get('start_time'),
+        'end': args.get('end_time')
+    }
+    device_os_version = argToList(args.get('device_os_version'))
+    policy_id = argToList(args.get('policy_id'))
+    alert_tag = argToList(args.get('alert_tag'))
+    alert_id = argToList(args.get('alert_id'))
+    device_username = argToList(args.get('device_username'))
+    device_id = argToList(args.get('device_id'))
+    device_os = argToList(args.get('device_os'))
+    process_sha256 = argToList(args.get('process_sha256'))
+    policy_name = argToList(args.get('policy_name'))
+    reputation = argToList(args.get('reputation'))
+    alert_type = argToList(args.get('alert_type'))
+    alert_category = argToList(args.get('alert_category'))
+    workflow = argToList(args.get('workflow'))
+    device_name = argToList(args.get('device_name'))
+    process_name = argToList(args.get('process_name'))
+    sort_field = args.get('sort_field')
+    sort_order = args.get('sort_order')
+    limit = args.get('limit')
+    contents = []
+
+    result = client.search_alerts_request(group_results, minimum_severity, create_time,
```

Comment:
please use keyword args, e.g. `group_results=group_results` to make sure we are passing the right args to where we want since there are a lot